pkg/products/threescale/prometheusRules.go

```diff
@@ -275,6 +275,19 @@ func (r *Reconciler) newAlertReconciler() resources.AlertReconciler {
 						For:    "15m",
 						Labels: map[string]string{"severity": "warning"},
 					},
+					{
+						Alert: "ThreescaleApicastWorkerRestart",
+						Annotations: map[string]string{
+							"summary":     "A new worker process in Nginx has been started",
+							"description": "A new thread has been started. This could indicate that a worker process has died due to the memory limits being exceeded. Please investigate the memory pressure on pod (instance {{ $labels.instance }})",
```

Comment:
Not a biggie, just that we can be more consistent with using the word "process" instead of "thread" here.

Reply:
@pepedocs this alert is a copy from an alert created in the 3scale operator https://github.com/3scale/3scale-operator/pull/484/files 

When we upgrade the 3scale operator to the version that hasthis alert, we'll remove the alert created by RHOAM.